ddhcpd/files/etc/ddhcpd.hook

```diff
@@ -1,7 +1,8 @@
 #!/bin/sh
 
-for i in /etc/ddhcpd.d/*
-do
-  $i "$@"
-done
-
+if [ -z "$(ls /etc/ddhcpd.d)" ]; then 
```

Comment:
do not parse the output of ls. Instead, please see here: https://stackoverflow.com/questions/4839214/does-ash-have-an-equivalent-to-bashs-nullglob-option

Reply:
the proper test would be for the executable flag.